WPMedia/fusion-news-theme-blocks
# THEMES 1203 V2 Short bio | Arc block - The author bio block needs styling for the social icons

**This PR goes with an `arc-themes-feature-pack` PR. You can find it [here](https://github.com/WPMedia/arc-themes-feature-pack/pull/442).**

## Description

Previously, the social icons in the `author-bio-block` (the short author bio) didn't have a set default fill color. The rest state was inherited from somewhere else. The purpose of this ticket was to add a proper rest state for the social icons. I made the following changes:
* I noticed that the `author-bio-block` has classes/selectors for hovering over the social icons and for the social links wrapper, but it didn't have a default class selector. I added that to `author-bio-block/_index.scss`
* I also added an `author-bio-social-link` section to `news.json`. I set the fill of the social icons to `$color-primary`.
* I updated the news styling in `./storybook/themes` to reflect the changes made to the block's `themes` directory
* As a result of these changes, some changes were also automatically made to the `site-styles` directory in `arc-themes-feature-pack`
    * You can find a link to that PR [here](https://github.com/WPMedia/arc-themes-feature-pack/pull/442)
    * These are needed to run this code

## Jira Ticket

[THEMES-1203](https://arcpublishing.atlassian.net/browse/THEMES-1203)

## Acceptance Criteria

1. Social media icons in the short bio block need a primary color alias token, like the full bio block

## Test Steps

1. Checkout this branch `git checkout THEMES-1203-author-bio-primary-color`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/author-bio-block`
3. Open your local PageBuilder instance, and create a new page
    * Select any layout you want. I used the right Right Rail
    * Place a Short Author Bio Block. No configurations need to be changed
    * Under the Global Content section of the Curate tab, set the Content Source to 'content-api', set the _id to '43UU6MCQERAMRPTD23B3CEXE7E' (you can use another if you want), then click 'Update'. Now, the Short Author Bio Block should be populated with the correct content
4. Verify that the social icons in the Short Author Bio Block are `$color-primary`(black right now) at rest and `$color-primary-hover` (`#384355` right now) when hovered over

## Effect Of Changes

This ticket should have no visual changes. Below is a video of how the short bio block should behave:

https://github.com/WPMedia/arc-themes-blocks/assets/103296586/c4fe1fa6-8cda-4909-8695-d00f896e6a70

## Dependencies or Side Effects

This PR is dependent on some changes to `arc-themes-feature-pack`. The PR is [here](https://github.com/WPMedia/arc-themes-feature-pack/pull/442).

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1203]: https://arcpublishing.atlassian.net/browse/THEMES-1203?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
